README.md

```diff
@@ -67,6 +67,10 @@ To disable parts of the functionality of this software, look no further than
 `config.json`. Here, in the sync section, you can disable the following things
 by setting them from `true` to `false` in a text editor:
 
+If the `config.json` is missing, copy it from `config.default.json`, the script
+will create `config.json` on first run, so this is only needed if you intend to
+customize before first run.
+
```

Comment:
I would not start with "_if the config.json is missing..._" because the user will think something is wrong.
Here is my version :
```
At first run, the script will create `config.json` based on `config.default.json`. If you want to customize settings before first run (ex. you don't want full sync) you can copy and edit `config.json` before launching the script.
```

Reply:
8481387

Reply:
Note that this need may go away in the future after command-line options support.

I already myself want to have `--no-tv` or `--sync=movies` (or `--movies`) option to indicate sync only certain library type.

Reply:
Great, you had a very good idea to add this click module. Seems very promising 👍 
I'll read other PR later.